core/vision/src/main/java/com/arcrobotics/ftclib/vision/UGBasicHighGoalPipeline.java

```diff
@@ -134,10 +143,12 @@ public boolean isBlueVisible() {
         return (blueRect != null);
     }
 
-    public static Optional<Point> getCenterofRect(Rect rect) {
+    public Point getCenterofRect(Rect rect) {
         if (rect == null) {
-            return Optional.empty();
+            return new Point(centerX, centerY);
         }
-        return Optional.of(new Point(rect.x + rect.width / 2, rect.y + rect.height / 2));
+        return new Point(rect.x + rect.width / 2, rect.y + rect.height / 2);
     }
-}
+
+
```

Comment:
Remove random whitespace but leave a single space between the last method closing brace and the closing class brace.
```java
public class MyClass {

  // stuff
    ...
  }  // closing brace of the last method

}